homeassistant/components/androidtv/const.py

```diff
@@ -0,0 +1,31 @@
+"""Android TV component constants."""
+DOMAIN = "androidtv"
+
+ANDROID_DEV = DOMAIN
+ANDROID_DEV_OPT = "androidtv_opt"
+
+CONF_ADB_KEY = "adb_key"
```

Comment:
I used "adbkey" because the key files are named `adbkey` and `adbkey.pub`. I would prefer to stick to that, and also make this constant `CONF_ADBKEY`.

Reply:
Reverted name as requested. BTW I was also breaking the YAML configuration.😊